rest/client.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"crypto/hmac"
 	"crypto/sha256"
+	_ "crypto/sha512"
```

Comment:
I think @lmars already knows about it. I believe I saw some code he wrote with this line before.

In case he doesn't it's well explained here: http://bridge.grumpy-troll.org/2014/05/golang-tls-comodo/. TL;DR is that we have an SSL certificate that is newer with a different encryption and this is to support this new SSL certificate.

On the other hand I may have not needed this line in `rest/rest_suite_test.go` and `test/support/test_app.go`. I'll crosscheck and remove if necessary.